src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaPlatformTests.cs

```diff
@@ -1,16 +1,42 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.DotNet.RemoteExecutor;
 using Xunit;
 
 namespace System.Security.Cryptography.SLHDsa.Tests
 {
-    public class SlhDsaPlatformTests : SlhDsaTestsBase
+    public class SlhDsaPlatformTests
     {
         [Fact]
-        public static void IsSupportedOnPlatform()
+        public static void IsSupported_AgreesWithPlatform()
         {
-            Assert.Equal(SupportedOnPlatform, SlhDsa.IsSupported);
+            Assert.Equal(PlatformSupportsSlhDsa(), SlhDsa.IsSupported);
+        }
+
+        private static bool PlatformSupportsSlhDsa()
+        {
+            if (PlatformDetection.IsOpenSsl3_5)
+            {
+                return true;
+            }
+
+            return false;
+        }
```

Comment:
(not sure if I also did that in ML-DSA) nit:

```suggestion
        private static bool PlatformSupportsSlhDsa() => PlatformDetection.IsOpenSsl3_5;
```